src/ui/DateTimePicker.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import moment from 'moment-timezone'
+import styled from 'styled-components'
+
+const CALENDAR_ICON_URL = 'https://svgshare.com/i/_5Y.svg'
```

Comment:
Should we use `fontawesome` instead?

Reply:
I tried, but since I'm passing an URL to the background CSS styling, I don't if it's possible. Anyway, I'll try it again.

Reply:
Basically, I can pass font-awesome only as content using the _:before_ selector, but not passing it directly to the background URL param, for this I need to pass an SVG instead.

So I decided to maintain the SVG hardcoded at least for now, and not access an external URL, since it can lead to security issues.